### PR TITLE
Add handoff support for code blob views

### PR DESCRIPTION
**Pull request by viktorasl**
I'm using handoff quite extensively (it's a great feature for me). Thus I did code blob view handoff just because sometimes it's easier to read code in a bigger screen.

There were no feature requests and from #331 I see that it's not expected that people will use handoff much. So if it looks like not a useful feature feel free to close it.

### CODE REVIEW

**Comment by Huddie** [comment]
@viktorasl Just to clarify, if your using Githawk you can handoff and open the issue/pr/whatever on github.com on your mac?

**Comment by viktorasl** [comment]
Yes, but probably the handoff can be done only to Safari. Issue/PR handoff was implemented with #331, this PR allows to open code blob view.

**Comment by j-f1** [comment]
Yeah, Handoff, as it works now, is great! Would like to see it everywhere though.

**Comment by viktorasl** [comment]
First of all, URL building for code blobs was already there and I didn't have to introduce `externalURL`.
Nontheless I've introduced `GithubURL` which should be used for concrete URL buildings. The idea is to move all Github URL building there & cover w/ tests.

File: Classes/Utility/GithubURL.swift [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+//  GithubURL.swift
+//  Freetime
+//
+//  Created by Viktoras Laukevicius on 20/02/2019.
+//  Copyright © 2019 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+enum GithubURL {
```
Comment (BasThomas) [inline Classes/Utility/GithubURL.swift]: Oh cool, I like this idea. :)

**Comment by BasThomas** [comment]
> Nontheless I've introduced `GithubURL` which should be used for concrete URL buildings. The idea is to move all Github URL building there & cover w/ tests.

Want to open an issue for that?

**Comment by viktorasl** [comment]
> > Nontheless I've introduced `GithubURL` which should be used for concrete URL buildings. The idea is to move all Github URL building there & cover w/ tests.
> 
> Want to open an issue for that?

Absolutely!

**Comment by viktorasl** [comment]
Raised an issue #2647;
Merged `master` & solved conflicts.

**Pull request merged**